src/encoded/static/components/navigation/components/LeftNav.js

```diff
@@ -12,6 +15,7 @@ import {
     BigDropdownPageTreeMenuIntroduction,
     BigDropdownBigLink
 } from './BigDropdown';
+import { SearchBar } from '.';
```

Comment:
Would write as `import { SearchBar } from './SearchBar';`
If are going to ".", then is going to "./index.js" to grab SearchBar from there, and such patterns can cause glitches in rare cases (i.e. LeftNav is trying to load file that exports LeftNav also; if LeftNav is imported somewhere before SearchBar, then it might throw an Exception).

Is also more performant to import direct files (rather than from index.js) where possible for performance, because it helps inform [WebPack Tree-Shaking](https://webpack.js.org/guides/tree-shaking/) to skip files that aren't imported at all. For example we do `import DropdownButton from 'react-bootstrap/esm/DropdownButton';` instead of `import DropdownButton from 'react-bootstrap';` now because it allows webpack to get rid of all of the React-Bootstrap components that we don't actually use from the compiled bundle.js/server.js.